examples/llm_plus_gnn/benchmark_model_archs.py

```diff
@@ -0,0 +1,38 @@
+# %%
+from g_retriever import benchmark_models, get_loss, inference_step
+
+from torch_geometric.datasets import UpdatedWebQSPDataset
+from torch_geometric.nn.models import GAT, MLP, GRetriever
+
+# %%
+hidden_channels = 1024
+num_gnn_layers = 1
+lr = 1e-5
+epochs = 2
+batch_size = 8
+eval_batch_size = 16
+
+# %%
+ds = UpdatedWebQSPDataset('benchmark_archs')
```

Comment:
once my work is merged in you should just rename this so you replace my work entirely